sycl/test-e2e/Matrix/element_wise_odd_sizes_bf16_impl.hpp

```diff
@@ -0,0 +1,107 @@
+#define SG_SZ 16
```

Comment:
Remove this define. It redefines what is defined in cpp files.

Reply:
Done